components/TaskCard.vue

```diff
@@ -134,8 +134,13 @@ export default {
     },
     checkForm(e) {
       e.preventDefault();
+      // Sanitize the user input by removing any HTML tags
+      const sanitizedInput = this.taskDescription.replace(/<[^>]+>/g, '');
 
-      if (this.taskDescription.length <= 0) {
+      // Set the sanitized input as the value of the input element
+      this.taskDescription = sanitizedInput;
```

Comment:
ditto: could be moved to mixin or similar

Reply:
Resolved